### ISSUE TITLE
The mfn factors model

### ISSUE THREAD

**Issue by jahdkaran** [oufuzhao/CLIB-FIQA#6]
Hi and thanks for the great work. 
You are using the mfn model for labeling the images (blur, luminance and expression). 
is this model trained by you? 
if not does it have a github link? 
the model has two other classifiers which you don't use. what are those?

**Comment by oufuzhao** [comment]
Hi, thanks for your interest.

We use the mfn train by myself to label the occlusion (not blur), illumination, and expression from the model prediction. 
The other two classifiers I do not use are blur and pose ( I'm a bit forgetful whether it's "invalid" or "pose", but it's most likely "pose") type. Because I used other methods to label these two factors, I didn't consider using them in the mfn model. However, when I was training the mfn model, I found that treating them together as multi-task training can make the validation loss decrease faster and better. Thus, I still retained these two unused classifiers in the final mfn model.

**Comment by jahdkaran** [comment]
Thanks for your response, 
Can you share the training procedure of the mfn factors model. (What is the dataset, what kind of preprocessing is needed and how to train the model)

**Comment by oufuzhao** [comment]
Hi, I am so sorry that I can't find the training code for the MNIST model anymore. But I can provide some training details for your information. The training dataset is Widerface. The approach of preprocessing is consistent with that of https://github.com/oufuzhao/CLIB-FIQA/blob/90f9ef30dc0efd7da4df8ed13f5ff9bc3cc36d29/generate_labels.py#L78. The training is in a multi-task form and uses CE loss on each subtask.
